data/json/items/tool_armor.json

```diff
@@ -590,6 +590,48 @@
     "covers": [ "HEAD" ],
     "magazine_well": 1
   },
+  {
+    "id": "wearable_atomic_light",
+    "type": "TOOL_ARMOR",
+    "category": "clothing",
+    "symbol": "[",
+    "color": "blue",
+    "looks_like": "survivor_light_on",
+    "name": "atomic headlamp",
+    "description": "This is a custom made reinforced headlamp powered by the magic of nuclear decay, focused for more usable brightness.  The adjustable strap allows it to be comfortably worn on your head or attached to your helmet.  Use it to close the cover and hide the light.",
+    "price": 4000,
+    "material": [ "steel", "plastic" ],
+    "flags": [ "LIGHT_200", "LEAK_DAM", "RADIOACTIVE", "DURABLE_MELEE", "OVERSIZE", "BELTED", "ALLOWS_NATURAL_ATTACKS" ],
+    "weight": "900 g",
```

Comment:
Does this thing's weight add up properly? It seems awfully lightweight.

Reply:
Actually, almost! I forgot to replace my eyeballed number with actual math. The total weight is 1kg, but the volume should be much larger, like 1.5L.